fe/fe-core/src/main/java/org/apache/doris/catalog/PartitionInfo.java

```diff
@@ -76,13 +76,16 @@ public class PartitionInfo implements Writable {
     // so we defer adding meta serialization until memory engine feature is more complete.
     protected Map<Long, TTabletType> idToTabletType;
 
+    protected Map<Long /* partition id */, Boolean /* mutable */> idToMutable;
```

Comment:
You can add this new properties in `DataProperty`

Reply:
good idea! so that we don't have to increase metadata version to  avoid metadata incompatible issues !